docs/source/examples/plots_examples.rst

```diff
@@ -1,6 +1,169 @@
 Plotting with manim
 =================================
 
+Examples to illustrate the use of GrapheScenes in Manim
+
+
+.. manim:: Plot1
+    :quality: medium
+    :save_last_frame:
+
+    class Plot1(GraphScene):
+        def construct(self):
+            self.setup_axes()
+            my_func = lambda x: np.sin(x)
+            func_graph=self.get_graph(my_func)
+            self.add(func_graph)
+
+.. manim:: Plot2yLabel
+    :quality: medium
+    :save_last_frame:
+
+    class Plot2yLabel(GraphScene):
+        CONFIG = {
+            "y_min": 0,
+            "y_max": 100,
+            "y_tick_frequency": 10,
+            "y_labeled_nums": np.arange(0, 100, 10)
+        }
+
+        def construct(self):
+            self.setup_axes(animate=True)
+            dot = Dot()
+            dot.move_to(self.coords_to_point(PI / 2, 20))
+            my_func = lambda x: 20 * np.sin(x)
+            func_graph = self.get_graph(my_func)
+            self.add(func_graph)
+            self.add(dot)
+
+.. manim:: Plot3DataPoints
+    :quality: medium
+    :save_last_frame:
+
+    class Plot3DataPoints(GraphScene):
+        CONFIG = {
+            "y_axis_label": r"Concentration[$\%$]",
+            "x_axis_label": "Time [s]",
+            }
+
+        def construct(self):
+            data = [1, 2, 2, 4, 4, 1, 3]
+            self.setup_axes(animate=True)
+            for time, dat in enumerate(data):
+                dot = Dot().move_to(self.coords_to_point(time, dat))
+                self.add(dot)
+                self.wait(1)
+                print(time)
+            self.wait()
+
+.. manim:: Plot3bGaussian
+    :quality: medium
+    :save_last_frame:
+
+    class Plot3bGaussian(GraphScene):
+        def construct(self):
+            def gaussian(x):
+                mu=3; sig= 1; amp=5
+                return amp*np.exp( ( -1/2 * ( (x-mu)/sig)**2 ) )
```

Comment:
This looks weird to me, and it's usually not recommended to use the semi-colon in python if it can be avoided.

Why not define `gaussian` outside of the class? And mu, sig, amp could be global variables as well. Right now, it kinda gives the (wrong) impression that `gaussian` needs to be defined inside `construct`.

Reply:
Ok, I will define it outside.
Sometimes I prefer it when I don't have to carry this self. in the code.
When everything is in "construct", it feels a little tighter to me.
But I can add another example, one where a function is defined in construct, and another one where the function is defined outside of the construct. 
In my computer class, I learned that we should best never use any global variables whenever they are not necessary.

Reply:
Woops sorry I missed this comment :smile: 

I agree that global constants are to be avoided (especially perhaps in python), but this seems like the right place for them. They are literally both *global* and *constant* (at least for the purposes of this example).